app/user_filters/models.py

```diff
@@ -0,0 +1,13 @@
+import uuid as uuidlib
+
+from django.contrib.auth.models import User
+from django.db import models
+from djsonb import fields as jfields
+
+
+class SavedFilter(models.Model):
+    """A filter saved by the user for later reference"""
+    uuid = models.UUIDField(primary_key=True, default=uuidlib.uuid4, editable=False)
+    label = models.CharField(max_length=255)
+    owner = models.ForeignKey(User)
+    filter = jfields.JsonField()
```

Comment:
Just going to note this: `filter` is a built-in function in Python, so this is going to override its definition and could potentially cause confusion if that function was going to be used in this scope. Odds of that happening are probably pretty low however.